integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testCaseBefore/MgwWithBackendTls.java

```diff
@@ -17,20 +17,25 @@
  */
 package org.wso2.choreo.connect.tests.testCaseBefore;
 
+import java.io.File;
 import org.testng.annotations.AfterTest;
 import org.testng.annotations.BeforeTest;
 import org.wso2.choreo.connect.tests.common.BaseTestCase;
 import org.wso2.choreo.connect.tests.context.MicroGWTestException;
 import org.wso2.choreo.connect.tests.util.ApictlUtils;
 
 import java.util.concurrent.TimeUnit;
+import org.wso2.choreo.connect.tests.util.TestConstant;
+import org.wso2.choreo.connect.tests.util.Utils;
 
 public class MgwWithBackendTls extends BaseTestCase {
 
     @BeforeTest(description = "initialise the setup")
     void start() throws Exception {
-        super.startMGW(null, true);
-
+        String targetDir = Utils.getTargetDirPath();
+        String confPath = targetDir + TestConstant.TEST_RESOURCES_PATH + File.separator + "cors" + File.separator
+                + "cors-disabled-config.toml";
```

Comment:
Do we have test cases for enabled scenario as well

Reply:
We do not have at the moment.

Reply:
First set run with global cors disabled (default conf before this PR) but API level cors enabled. (This is the one we are talking about here)
Second set is where API Level Cors is disabled but global cors enabled. Hence the CORS configuration governed by global config and it is enabled overall.
So I think it is enough.